eng/scripts/MavenPackaging/MavenPackaging.psd1

```diff
@@ -0,0 +1,18 @@
+@{
```

Comment:
Unless there is a good reason for this to be a PS module I'd suggest just having it be a helper script that we dot source into places that need it. PS modules end up causing more trouble then they are worth in a lot of our usages as such @chidozieononiwu is actually starting to move us away from them.

Reply:
What kind of problems? Curious now.

Reply:
I've moved back to dot-sourcing for now.

Reply:
@mitchdenny, if you recall a couple of weeks back, there were a bunch of [transient errors installing PS modules that Dozie had to fix](https://github.com/Azure/azure-sdk-tools/pull/781).

Reply:
Yes there are various transient networking errors that we can completely avoided if we use all local scripts. Even using local modules things are a little difficult to debug and test locally as you always have to remove the module after you make an updated change to it before you can reinstall it. There have also been other odd issues like we had to disable the progress dialog in various places when installing even local modules. There were enough small issues that I felt we weren't getting anything useful from PS modules so I've decided to just use raw scripts.